### PR TITLE
Add Flux 0.88.0 release notes

### LINKED ISSUE #1599: Update Flux 0.88.0 release notes
Update Flux release notes up to the Flux .88.0 release.

https://github.com/influxdata/flux/releases

### PR DESCRIPTION
**Pull request by pierwill**
Closes #1599.

Also, add v.85.0, .86.0, .87.0, and .87.1 release notes; fix formatting errors in previous release notes.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 1]
```diff
@@ -8,6 +8,65 @@ menu:
...
+- Fix compiler type inference with extended records.
+- Colm Flux grammar updates, keywords, string interpolation, UTF8 IDs.
+- Exponent operator have higher precedence.
+
+---
+
+## v0.86.0 [2020-09-21]
+
+### Features
+- Add operator profiler.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/flux.md:49]: Curious, would it be helpful to say anything more about profiler?

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 2]
```diff
@@ -8,6 +8,65 @@ menu:
...
+- Add duration conversion.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/flux.md:50]: should we add supported types to the duration() function?

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 3]
```diff
@@ -8,6 +8,65 @@ menu:
...
+- Add naive bayes classification.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/flux.md:51]: should naiveBayesClassifier be added to Flux docs?

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 4]
```diff
@@ -8,6 +8,65 @@ menu:
...
+- Reset pointer after scanning invalid Unicode.
+- Catch references to non-existent columns.
+- Propagate span context to `source.Run`.
+
+---
+
+## v0.85.0 [2020-09-14]
+
+### Features
+- Add `Aggregate.window` for an alternative windowing aggregate.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/flux.md:63]: Add to [aggregateWindow() function](https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/aggregatewindow/)?

**Review by kelseiv — COMMENTED** [review]
Couple questions/comments that may already be addressed in other docs

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 5]
```diff
@@ -8,6 +8,65 @@ menu:
...
 
+## v0.88.0 [2020-10-05]
+
+### Features
+- Moves functions from `v1` package to `schema` package.
+
+### Bug fixes
+- Fix field type error in test.
+- Update buildinfo script to handle new and deleted files.
+- Sets default quantile method when not specified.
```
[the remark this answers is not in the document]
Comment (pierwill) [reply content/influxdb/v2.0/reference/release-notes/flux.md:19]: 👍

**Pull request merged**